cmd/swarmctl/service/flagparser/port.go

```diff
@@ -30,6 +30,9 @@ func parsePorts(flags *pflag.FlagSet, spec *api.ServiceSpec) error {
 			Protocol:      protocol,
 			TargetPort:    port,
 			PublishedPort: swarmPort,
+			// In swarmctl all ports are by default
+			// PublishModeHost
+			PublishMode: api.PublishModeHost,
```

Comment:
Seems like a pretty big behavioral change. Isn't it worth adding a flag to control this?